.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npx pretty-quick --staged
```

Comment:
pretty-quick seems a bit abandoned (last release 2021), don't we have any other more popular tool ?
Why not using something that others use, like:
https://github.com/angular/angular-cli/blob/main/.husky/pre-commit
https://github.com/ant-design/ant-design/blob/master/.husky/pre-commit